fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraLdp.java

```diff
@@ -770,6 +789,22 @@ private String mintNewPid(final String slug) {
         return pid;
     }
 
+    private String handleWantDigestHeader(final FedoraBinary binary, final String wantDigest)
+            throws UnsupportedAlgorithmException {
+        // handle the Want-Digest header with fixity check
+        final Collection<String> preferredDigests = parseWantDigestHeader(wantDigest);
+        if (preferredDigests.isEmpty()) {
+            LOGGER.warn("Unsupported digest algorithm provided in 'Want-Digest' header: {}", wantDigest);
```

Comment:
`warn` seems to be too high of a log level.  The exception will already be logged at the `DEBUG` level by the exception handler for `UnsupportedAlgorithmException`, so I think you can just remove this

Reply:
Sure, the warn logging is removed. See commit c306e2f.